python/ava-common.py

```diff
@@ -73,12 +73,13 @@ def run(command, **kwargs):
     del p
     if gnu_time:
         # make sure the stats are there
-        GNU_TIME_BYTES = b"\n\tUser time"
-        assert GNU_TIME_BYTES in err, "error: cannot find GNU time info in stderr"
-        gnu_time_str = err[err.index(GNU_TIME_BYTES) :].decode("ascii")
-        gnu_time_stats = gnu_time_parse(gnu_time_str, logfd, debug)
-        err = err[0 : err.index(GNU_TIME_BYTES) :]
-        return returncode, out, err, gnu_time_stats
+        if b"User time" in err:
```

Comment:
If you want this not to assert, then use the GNU_TIME_BYTES variable (instead of the b"User time" binary string) to check, and then remove the assert line

Reply:
Oops. I'll fix.